tutorials/evoked/30_eeg_erp.py

```diff
@@ -373,7 +374,204 @@
+# Instead of a peak, we could just measure the rising edge of a peak, for
+# instance, which is not ideal. The following demonstrates why visual
+# inspection is crucial. Below, we use a known bad time window (.09 to .12
+# seconds) to search for a peak on ``eeg59``.
+
+# Get BAD peak measures
+bad_tmin, bad_tmax = .09, .12
+_, bad_lat_roi, bad_amp_roi = \
+    l_vis_roi.get_peak(mode='pos', tmin=bad_tmin, tmax=bad_tmax,
+                       return_amplitude=True)
+
+# Convert latency and amplitude to msec and microvolts
+bad_lat_roi *= 1e3
+bad_amp_roi *= 1e6
+
+# Print output
+print('** PEAK MEASURES FROM A BAD TIME WINDOW **')
+print('Channel: eeg59')
+print(f'Peak Latency: {bad_lat_roi:.3f} msec')
+print(f'Peak Amplitude: {bad_amp_roi:.3f} \u00B5V')
+
+# %%
+# If all we had were the above values, it would be unclear if they are truly
+# identifying peak or just a the rising edge. However, it becomes clear that
+# the .09 to .12 second time window use for the search is missing the peak
+# on ``eeg59``. This is shown in the bottom panel where we see the bad time
+# window (highlighted in orange) misses the peak (the pink star). In contrast,
+# the time window defined initially (.065 to .115 seconds; highlighted in blue)
+# returns the actual peak instead of a value on the rising edge. Visual
+# inspection will always help you to convince yourself the data returned are
+# really peaks.
+
+# Make an empty figure handle and axis
+fig, (ax1, ax2) = plt.subplots(nrows=2, ncols=1)
+
+# Plot the ERP, actual peak, and the good time window searched
+l_vis_roi.plot(axes=ax1, time_unit='ms', show=False,
+               titles='Bad time window missing peak')
+ax1.plot(lat_roi, amp_roi, marker="*", color='C6')
+ax1.axvspan(bad_tmin * 1e3, bad_tmax * 1e3, facecolor='C1',
+            alpha=.3)
+ax1.set_xlim(-50, 150)  # Show zoomed in around peak
+
+# Plot the ERP, actual peak, and the bad time window searched
+l_vis_roi.plot(axes=ax2, time_unit='ms', show=False,
+               titles='Good time window finding peak')
+ax2.plot(lat_roi, amp_roi, marker="*", color='C6')
+ax2.axvspan(tmin * 1e3, tmax * 1e3, facecolor='C0',
+            alpha=.3)
+ax2.set_xlim(-50, 150)  # Show zoomed in around peak
+plt.tight_layout()
+
+# %%
+# Mean Amplitude
+# --------------
+#
+# Another common practice in ERP studies is to define a component (or effect)
+# as the mean amplitude within a specified time window. One advantage of this
+# approach is that it is less sensitive to high frequency noise (compared to
+# peak amplitude measures) because averaging over a time window is, in essence,
+# a filter.
+#
+# When using mean amplitude measures, it is a good idea to have a predefined
+# time window for extracting mean amplitude. Selecting the time window based
+# on the observed data (e.g., the grand average) can inflate false positives in
+# ERP research :footcite:`LuckGaspelin2017`.
+#
+# Below, demonstrates how to pull out the mean amplitude between .065 sec and
```

Comment:
```suggestion
# The following examples demonstrates how to pull out the mean amplitude between .065 s and
```

Reply:
case mismatch in @cbrnr's suggestion, should be "The following examples demonstrate"

Reply:
Or maybe rather "The following example demonstrates..."?